### ISSUE TITLE
Tool occasionally causes problems with glyph editor and font overview panels

### ISSUE THREAD

**Issue by liamspradlin** [jtanadi/CheckParallelTool#3]
Occasionally, using Robofont with Check Parallel Tool installed will cause problems with the glyph editor and font overview panels and their relationship to one another. Typically, this means...

- Selecting things in the glyph editor is either limited or impossible (for instance cmd+a will select all glyphs in the font overview, rather than the contours in the glyph editor) and 
- Copying/pasting will apply not to the content of the glyph editor but rather the currently selected glyph in the font overview.

Typically the issue takes a while to manifest and it seems to me (from experiencing this several times) that the issue is triggered by opening multiple UFO files, one after the other. Sometimes just having a file open and opening another will trigger the issue.

Uninstalling Check Parallel Tool and restarting Robofont solves the issue. I've attached a screenshot here to show the other extensions I normally have installed.

<img width="342" alt="Screen Shot 2019-11-09 at 10 18 26 PM" src="https://user-images.githubusercontent.com/10691248/68538224-e569e700-033e-11ea-90bb-f78cf5b9e4b3.png">

**Comment by typemytype** [comment]
the problem is that the extension is reusing the same `guideStatus` for each open window. This is not allowed as a view can only be added once to a window (or a vanilla.Group).

https://github.com/jtanadi/CheckParallelTool/blob/master/CheckParallelTool.roboFontExt/lib/checkParallel.py#L52-L53

you need to create a new `statusGuide` view for each glyph window, and store it in a `dict`-map with the glyph window as key and the `statusGuide` as value, so you can retrieve the status view for the current window

good luck!

**Comment by jtanadi** [comment]
@typemytype that makes sense, thank you!

**Comment by gupta-vidit** [comment]
@jtanadi Did you get a chance to fix this?

**Comment by jtanadi** [comment]
@gupta-vidit No, I've been out of the font loop for quite some time now, and I don't think I will be able to pick it back up any time soon. I should really archive the project at this point.
